unit.go

```diff
@@ -0,0 +1,338 @@
+// Copyright ©2013 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package unit
+
+import "fmt"
+
+// Unit is a type a value with generic SI units. Most useful for
```

Comment:
`s/Unit is a type a value/Unit describes a value in generic SI units/`
or something along these lines?
(not a native english speaker, but I had issues parsing that sentence)